app/models/solidus_paypal_braintree/configuration.rb

```diff
@@ -11,9 +11,7 @@ class Configuration < ::Spree::Base
       messaging: { availables: %w[true false], default: 'false' }
     }.freeze
 
-    unless respond_to?(:preference)
```

Comment:
@MinasMazar any chance you recall why we added this check [here](https://github.com/solidusio/solidus_paypal_braintree/commit/128ddddd26fd6146f82ad785f1eb48404fbc5b7a)? 🙏

Reply:
@kennyadsl I'm sorry, but I'm not aware right now. I think we can remove it.